library/Fum2GitHub/Fum.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE OverloadedStrings #-}
 module Fum2GitHub.Fum (
+    AuthToken(AuthToken),
```

Comment:
it's easier to export with `AuthToken(..)` which states *export all constructors* (it will export `getAuthToken` too)